app/services/ipam/db.rb

```diff
@@ -8,7 +8,7 @@ def suggest_ip
       to = subnet.to.present? ? IPAddr.new(subnet.to) : IPAddr.new(subnet_range.last.to_i - 2, subnet.family)
       (from..to).each do |address|
         ip = address.to_s
-        if !subnet.known_ips.include?(ip) && !excluded_ips.include?(ip)
+        if !subnet.known_ips.include?(ip) && !(excluded_ips || []).include?(ip)
```

Comment:
You might want to fix this globally, i.e. for all providers here: https://github.com/theforeman/foreman/blob/79c0664a008afe0fea54a0b5256426f3867a02bc/app/services/ipam/base.rb#L13